viz/cmd/install.go

```diff
@@ -93,7 +76,7 @@ A full list of configurable values can be found at https://www.github.com/linker
 
 	cmd.Flags().BoolVar(
 		&skipChecks, "skip-checks", false,
-		`Skip checks for namespace existence`,
+		`Skip checks for linkerd core control-plane existence`,
```

Comment:
code formatting nit: it seems to me that all other command flags are written out on a single line, however, this file has its command line flags spanning 3 lines. Should we make sure that this file matches up with other cmd files?

Reply:
Makes sense. Updating...